perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -58,6 +64,9 @@ def __init__(self, vm_spec):
         self.user_name = self.DEFAULT_USERNAME
         self.boot_wait_time = None
         self.image = self.image or self.DEFAULT_IMAGE
+        self.boot_device = None
+        self.root_disk_allocated = False
```

Comment:
Use of "boot" and "root" seems inconsistent here. If they mean the same thing, please standardize on one of them.